### ISSUE TITLE
3rd Party component in SmartDots

### ISSUE THREAD

**Issue by neil-ices-dk** [ices-eg/SmartDots#36]
The new compiled version of SmartDots has a 3rd part component that needs to be licenced/paid. Is it possible to strip this out and replicate the functionality with standard code/another free solution. To be discussed if possible?

**Comment by wimallegaert** [comment]
That's correct, SmartDots uses controls of the DevExpress library (https://www.devexpress.com/products/net/controls/wpf/). The DevExpress library has been integrated since version 1.0. 

Most notable DevExpress controls we use in SmartDots are the Grids, the Docking Panels and the styles. Compared with the standard controls the DevExpress Docking Panels and Grids are more powerfull.

It is possible to replace these controls, but stripping these controls out of SmartDots will take quite a lot of effort. Buying a license will be a cheaper solution ($899,99).

Nevertheless, I understand your question. If we would have known from the beginning that SmartDots would become an open solution, we would have used more standard controls.

**Comment by neil-ices-dk** [comment]
thanks Wim - looks like it is used extensively and i completely understand why you went down this route in terms of speed of deployment etc. So in the short term it's not an issue as the most likely users (ourselves and the big institutes won't have a problem). So maybe it's a wishlist item to ponder - 'SmartDots  Lite' for instance ;)
